### DISCUSSION TITLE
When would there be a stable version ?

### DISCUSSION THREAD

**Discussion by hi-glenn** [databendlabs/databend#13819] [Q&A] [answered]
As the title

**Comment by bohutang** [comment, marked as answer]
Thanks for asking, this is a great question :)

We're planning to release a stable v1.3 by late this year or early 2024.

Until then, we encourage to utilize Databend [releases](https://github.com/datafuselabs/databend/releases), which has been rigorously tested to guarantee smooth version upgrades without any performance setbacks or compatibility issues.
